typespec-extension/src/code-model-builder.ts

```diff
@@ -181,6 +181,7 @@ import {
 } from "./operation-utils.js";
 import { isArmCommonType } from "./type-utils.js";
 import pkg from "lodash";
+import { log } from "console";
```

Comment:
log not used?

Reply:
No, removed. It's added accidentally by using IDE...